custom_components/tuya_local/devices/forceclima_soundless.yaml

```diff
@@ -28,21 +28,15 @@ primary_entity:
       name: temperature
       range:
         min: 16
-        max: 30
-      mapping:
-        - scale: 0
-          step: 1
+        max: 31
       unit: C
 
     - id: 3
       type: integer
       name: current_temperature
       range:
```

Comment:
Range will be ignored for current_temperature anyway, as it is read-only so will just follow what the device sends. But I think temperatures above and below the settable range can probably still be read from the sensor.